### PR TITLE
Yellowstone za

### PR DESCRIPTION
**Pull request by zhannaavanesova**

### CODE REVIEW

File: bxhealth/src/pages/manage/message-doctor.js [hunk 1]
```diff
@@ -18,6 +19,8 @@ import "../../styles/pages/dashboard.scss";
 
 class Dashboard extends React.Component {
   render() {
+    const session = new Session();
```
Comment (michaelsanchez-pingidentity) [inline bxhealth/src/pages/manage/message-doctor.js:22]: Is there a reason you're instantiating a new session object inside render()? Everytime state changes, the render() method is executed, whcih means we're creating another session object instance.

**Review by michaelsanchez-pingidentity — CHANGES_REQUESTED** [review]
See inline comment.

File: bxhealth/src/pages/manage/message-doctor.js (hunk 1 above)
Comment (zhannaavanesova) [reply bxhealth/src/pages/manage/message-doctor.js:22]: I did this so I could use getAuthenticatedUserItem("patientFullName", "session") in line 42. Is there a better way to do this?

Comment (michaelsanchez-pingidentity) [reply bxhealth/src/pages/manage/message-doctor.js]: You should instantiate new objects in the constructor. It gets created with the instance of the class (so only once)  and is available in the *this* scope, so it's globally available.

**Review by michaelsanchez-pingidentity — APPROVED** [review]
Reviewed

**Pull request merged**